### PR TITLE
postmoogle: init at 0.9.21

### PR DESCRIPTION
**Pull request by amuckstot30**
## Description of changes

https://gitlab.com/etke.cc/postmoogle
https://github.com/etkecc/postmoogle

## Things done

- Built on platform(s)
  - [x] x86_64-linux
  - [x] aarch64-linux
  - [ ] x86_64-darwin
  - [ ] aarch64-darwin
- For non-Linux: Is sandboxing enabled in `nix.conf`? (See [Nix manual](https://nixos.org/manual/nix/stable/command-ref/conf-file.html))
  - [ ] `sandbox = relaxed`
  - [ ] `sandbox = true`
- [x] Tested, as applicable:
  - [NixOS test(s)](https://nixos.org/manual/nixos/unstable/index.html#sec-nixos-tests) (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
  - and/or [package tests](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#package-tests)
  - or, for functions and "core" functionality, tests in [lib/tests](https://github.com/NixOS/nixpkgs/blob/master/lib/tests) or [pkgs/test](https://github.com/NixOS/nixpkgs/blob/master/pkgs/test)
  - made sure NixOS tests are [linked](https://github.com/NixOS/nixpkgs/blob/master/pkgs/README.md#linking-nixos-module-tests-to-a-package) to the relevant packages
- [ ] Tested compilation of all packages that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review rev HEAD"`. Note: all changes have to be committed, also see [nixpkgs-review usage](https://github.com/Mic92/nixpkgs-review#usage)
- [x] Tested basic functionality of all binary files (usually in `./result/bin/`)
- [24.11 Release Notes](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2411.section.md) (or backporting [23.11](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2311.section.md) and [24.05](https://github.com/NixOS/nixpkgs/blob/master/nixos/doc/manual/release-notes/rl-2405.section.md) Release notes)
  - [ ] (Package updates) Added a release notes entry if the change is major or breaking
  - [ ] (Module updates) Added a release notes entry if the change is significant
  - [ ] (Module addition) Added a release notes entry if adding a new NixOS module
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/CONTRIBUTING.md).

---

Add a :+1: [reaction] to [pull requests you find important].

[reaction]: https://github.blog/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/
[pull requests you find important]: https://github.com/NixOS/nixpkgs/pulls?q=is%3Aopen+sort%3Areactions-%2B1-desc

### CODE REVIEW

**Comment by amuckstot30** [comment]
It's my first time ever contributing (if you can even call it that) to any open source project, so please tell me if there's anything I could've done better

File: pkgs/by-name/po/postmoogle/package.nix [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+  version = "0.9.21";
+
+  src = fetchFromGitHub {
+    owner = "etkecc";
+    repo = "postmoogle";
+    rev = "v${version}";
+    hash = "sha256-/AuxrIvxoKb08uf4EOYXorl7vJ99KgEH9DZYLidDzI4=";
+  };
+
+  CGO_ENABLED = 0;
```
[the remark this answers is not in the document]
Comment (amuckstot30) [reply pkgs/by-name/po/postmoogle/package.nix:17]: Could you please clarify? The justfile for postmoogle also disables cgo. Enabling cgo only introduces an additional store path for the C linker in the binary. I haven't seen any other package definitions mention why they're disabling cgo.

**Comment by pbsds** [comment]
runs fine

**Comment by pbsds** [comment]
Thank you and welcome to nixpkgs!

Now that you've added yourself to `maintainer-list.nix` you should in short time be invited to join the `nixos/nixpkgs-maintainers` group. That invite email will expire in a week. Joining the group will give you additional permissions in nixpkgs, like the ability to change labels on issues and pull requests.

**Pull request merged**